ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/transaction/ConsensualWireTransaction.kt

```diff
@@ -0,0 +1,24 @@
+package net.corda.v5.ledger.consensual.transaction
+
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.DoNotImplement
+import net.corda.v5.crypto.SecureHash
+import net.corda.v5.crypto.merkle.HASH_DIGEST_PROVIDER_TWEAKABLE_NAME
+import net.corda.v5.ledger.common.transactions.PrivacySalt
+import net.corda.v5.ledger.consensual.ConsensualState
+import java.security.PublicKey
+import java.time.Instant
+
+@DoNotImplement
+@CordaSerializable
+interface ConsensualWireTransaction {
```

Comment:
I am not quite sure of the value of this interface. We do need a WireTransaction that holds a PrivacySalt and a List<List<ByteArray>>, however it is largely an internal concern of serialization and storage. This interface is presumably meant for informational purposes in which case it should be talking in terms of concrete information e.g. Class<*> for state types and deserialized PublicKeys for signers and states, but that blurs into whatever the 'LedgerTransaction' type is so maybe we don't need to expose this.

Reply:
I have removed this.